src/accelerate/accelerator.py

```diff
@@ -231,6 +255,70 @@ def prepare_model(self, model):
             model.forward = torch.cuda.amp.autocast()(model.forward)
         return model
 
+    def _prepare_deepspeed(self, *args):
+
+        ds_plugin = self.state.deepspeed_plugin
+        self.ds_config = ds_plugin.ds_config
```

Comment:
Can we say `self.deepspeed_config` to go with `self.state.deepspeed_plugin`?

Also would love if this function was using `deepspeed_plugin` instead of `ds_pluging`.

Reply:
I didn't understand the question. Could you please clarify??

I will rename `ds_plugin` to `deepspeed_plugin`.

Reply:
I would prefer we use `deepspeed_plugin` and `deepspeed_config` instead of `ds_plugin` and `ds_config`